examples/native/native.rs

```diff
@@ -114,9 +109,10 @@ fn main() -> anyhow::Result<()> {
         let key = include_bytes!(concat!(env!("OUT_DIR"), "/key.key"));
         NativeTlsServerConfig::new(cert.to_vec(), key.to_vec())
     };
-    let mut cloud_cfg = CloudConfig::new(ROBOT_NAME, LOCAL_FQDN, FQDN, ROBOT_ID, ROBOT_SECRET);
-    cloud_cfg.set_tls_config(cfg);
-    let esp32_srv = NativeServer::new(robot, cloud_cfg);
-    esp32_srv.start(ip)?;
+
+    let app_config = AppClientConfig::new("".to_string(), "".to_string(), ip, "".to_owned());
```

Comment:
might be good to take these from env variables and unwrap_or to empy strings later